src/components/ListboxGrid/ListboxGrid.tsx

```diff
@@ -62,18 +65,18 @@ export default function ListboxGrid(props: ListboxGridProps) {
     }
   }, []);
 
-  const handleOnFullscreen = () => {
-    const { model } = store.getState();
-    if (model) {
-      onFullscreen?.(model.id);
-    }
-  };
+  useEffect(() => {
+    setDisableExpandButton(!!constraints?.active);
+  }, [constraints?.active]);
```

Comment:
Can we put `constraints?.active` directly into the returned HTML and get rid of the `useEffect`?

Reply:
Yes, nice